website/src/components/markdown-layout.js

```diff
@@ -49,7 +208,16 @@ const components = {
   ul: Ul,
   ol: Ol,
   li: Li,
-  table: Table,
+  // eslint-disable-next-line react/display-name
+  table: props => (
+    <div
+      css={css`
+        overflow: auto;
```

Comment:
This is to help "seeing" the table in case the responsive breakpoint hasn't kicked in yet and the table cannot reduce its size anymore

![2019-04-29 14 35 14](https://user-images.githubusercontent.com/1110551/56896461-13b36380-6a8c-11e9-974f-19ecf2ff0da7.gif)